internal/transform/config_setting.go

```diff
@@ -0,0 +1,161 @@
+package transform
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/stellar/go/ingest"
+	"github.com/stellar/go/xdr"
+	"github.com/stellar/stellar-etl/internal/utils"
+)
+
+// TransformConfigSetting converts an config setting ledger change entry into a form suitable for BigQuery
+func TransformConfigSetting(ledgerChange ingest.Change) (ConfigSettingOutput, error) {
+	ledgerEntry, changeType, outputDeleted, err := utils.ExtractEntryFromChange(ledgerChange)
```

Comment:
Do we also need to add a switch statement in `ExtractEntryFromChange` to handle evicted (expired?) entries? If that `LedgerEntryChangeType` is different from deleted, I want to make sure we're handling it.

Reply:
Currently there is no evicted/expired state (at least from looking at the XDR). When it's added we can update this to handle it